enterprise/server/test/integration/ci_runner/ci_runner_test.go

```diff
@@ -748,7 +748,7 @@ actions:
 	assert.Contains(t, result.Output, "args: {{ Switcheroo! }}")
 }
 
-func TestRunAction_TargetRepoOnly(t *testing.T) {
+func TestRunAction_PushedRepoOnly(t *testing.T) {
```

Comment:
The workflow service currently sets both pushed and target on push events, so we should probably set both flags here to make sure we're testing the actual usage?

(If the runner service only sets pushed repo URL, then we could make a copy of this test to make sure it works in both cases)

```suggestion
func TestRunAction_PushEvent(t *testing.T) {
```

Reply:
Added a new test